modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -79,17 +102,31 @@ one IP address pool. If you need to access the services from an external network
 configure load balancers and routers to manage the traffic.
 <10> You must set the platform to `none`. You cannot provide additional platform
 configuration variables for bare metal infrastructure.
+ifndef::disconnected[]
 <11> The pull secret that you obtained from the
 link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers]
 page. This pull secret allows you to authenticate with the services that are
 provided by the included authorities, including Quay.io, which serves the
 container images for {product-title} components.
+endif::disconnected[]
+ifdef::disconnected[]
+<11> For `bastion_host_name`, specify the registry domain name
+that you specified in the certificate for your mirror registry, and for
+`<credentials>`, specify the base64-encoded user name and password for
+your mirror registry.
+endif::disconnected[]
 <12> The public portion of the default SSH key for the `core` user in
 {op-system-first}.
 +
 [NOTE]
 ====
 For production {product-title} clusters on which you want to perform installation
 debugging or disaster recovery, you must provide an SSH key that your `ssh-agent`
-process uses to the installation program.
+process uses to the installer.
```

Comment:
Backsliding?  I thought we preferred "installation program" here (e.g. [here][1]).

[1]: https://github.com/openshift/openshift-docs/commit/ac3ee1512d2a27dbe35a12ea1c04190b02a4523a#diff-8e817d501daab6331bc113b6b93cc701L9